CodeEdit/Features/NavigatorArea/FindNavigator/FindNavigatorResultList/FindNavigatorListViewController.swift

```diff
@@ -231,6 +231,17 @@ extension FindNavigatorListViewController: NSOutlineViewDelegate {
         outlineView.noteHeightOfRows(withIndexesChanged: indexes)
     }
 
+    // swiftlint:disable:next function_parameter_count
+    func outlineView(
+        _ outlineView: NSOutlineView,
+        toolTipFor cell: NSCell,
+        rect: NSRectPointer,
+        tableColumn: NSTableColumn?,
+        item: Any,
+        mouseLocation: NSPoint
+    ) -> String {
+        return ""
+    }
```

Comment:
I’m a bit confused about how this function works. Can you help me out? 😅

Reply:
I think it's a placeholder. but then we can better add `// TODO: Needs to be implemented` or something.